styles.css

```diff
@@ -0,0 +1,541 @@
+}
+
+.contant {
+  flex-direction: column;
+  gap: 24px;
+  justify-content: center;
+  height: 100vh;
+}
+
+h1 {
+  font-weight: 1300;
+  font-size: 70px;
+  line-height: 70px;
+  text-align: center;
+  letter-spacing: 0.37px;
+  color: var(--Primary);
+}
+
+h2 {
+  font-weight: 1300;
+  font-size: 70px;
+  line-height: 70px;
+  text-align: center;
+  letter-spacing: 0.37px;
+  color: var(--Secondary);
+}
+
+p {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  color: var(--Primary);
+  flex: none;
+  flex-grow: 0;
+  max-width: 750px;
+}
+
+.social {
+  gap: 20px;
+  margin-top: 3%;
+}
+
+.next {
+  margin-top: 5%;
+  display: flex;
+}
+
+.head {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: var(--Primary);
+}
+
+.img-multi {
+  background-image: url(assest/img/recent/holder.png);
+  background-size: cover;
+  height: 260px;
+  width: 100%;
+  margin-bottom: 20px;
+}
+
+.cards {
+  display: grid;
+  align-items: center;
+  gap: 1.5rem;
+  padding: 1.5rem 0;
+  margin: 0 1rem;
+  grid-template-columns: auto auto auto;
+}
+
+.card {
+  height: 474px;
+  width: 327px;
+  flex-direction: column;
+  border: 1px solid #ebf0ee;
+  border-radius: 10px;
+}
+
+.text {
+  margin-bottom: 25px;
+  flex-direction: column;
+}
+
+h3 {
+  margin-bottom: 5%;
+}
+
+.tags ul {
+  flex-direction: row;
+  list-style: none;
+  column-gap: 5px;
+  padding: 5px;
+  margin-bottom: 20px;
+}
+
+.tag {
+  background: #ebf0ee;
+  padding: 5px;
+  border: 1px solid #fff;
+  margin-bottom: 5px;
+  color: #000;
+  border-radius: 5px;
+  font-size: 12px;
+  line-height: 16px;
+  font-weight: 600;
+}
+
+.see-project {
+  background: var(--Secondary);
+  border: none;
+  color: #fff;
+  border-radius: 5px;
+  padding: 5px 12px;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+}
+
+.see-project:hover {
+  background-color: #36b37f;
+  cursor: pointer;
+}
+
+.contant-1 {
+  flex-direction: column;
+  position: relative;
+  gap: 24px;
+  text-align: center;
+}
+
+.get {
+  background: #36b37f;
+  border: none;
+  padding: 9px 19px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  border-radius: 5px;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  color: white;
+}
+
+.tager {
+  display: grid;
+  align-items: center;
+  gap: 1.5rem;
+  padding: 1.5rem 0;
+  margin: 0 1rem;
+  grid-template-columns: auto auto auto;
+  justify-content: center;
+}
+
+.card-about {
+  background-color: #ebf0ee;
+  height: 352px;
+  width: 369px;
+  flex-direction: column;
+  border: 1px solid black;
+  border-radius: 10px;
+}
+
+.lang {
+  height: 70px;
+  width: 70px;
+  background-color: #fff;
+  border-color: 1px rgba(255, 255, 255, 0.363);
+  padding: 10px;
+  border-radius: 100px;
+  margin-top: 15%;
+}
+
+.text-1 {
+  margin-bottom: 120px;
+  flex-direction: column;
+}
+
+h4 {
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+}
```

Comment:
- [ ] The `color` of the titles for each card is `#344563`. Kindly change the current color you have so that your design looks identical to that on Figma

### Your Design

![image](https://user-images.githubusercontent.com/77106962/219493812-e99f47f2-4ee9-4b44-a73f-2f808baf6471.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/219493891-88515a12-e72f-48f4-b76d-28d5034f0aa3.png)